### ISSUE TITLE
Web Components section: proposals section

### ISSUE THREAD

**Issue by web-padawan** [andreasbm/web-skills#21]
FIrst of all, thanks for the great project! Let me suggest some ideas for Web Components part:

1. Constructible Stylesheets lack consensus at the moment, and are effectively Chrome-only. So, I would recommend to either remove them until consensus is reached, or downplay.

2. Proposals section is lacking. IfMO, the following items would make sense to be listed there:
  - Form Participation
  - Custom `:state` pseudo class
  - AOM 
  - Constructible Stylesheets

**Comment by andreasbm** [comment]
Hi Serhii - thanks for opening this issue, I'm glad that you like the project! Great suggestion, I will see if I can make these changes later today. I'm thinking of one of two solutions:

1. Remove the "Constructible Stylesheets" and "Form Participation" skills and create a skill called "Proposals" that contains links to the proposals. 

2. Create a skill called "Proposals" with sub-skills for some of the proposals. 

I like the simplicity of solution #1 but think I prefer solution #2 because it might engage people more and make them explore the various proposals. What are your thoughts on this?

**Comment by web-padawan** [comment]
Proposals skill (separate from web components) make a lot of sense to me in general. 
There are various APIs not implemented by all browsers which are worth trying.

However, I would recommend to not add any proposals blocked by any vendors, e.g. 

- Native File System API (Mozilla's position is to "defer")
- Constructible Stylesheets (blocked by WebKit)
- `beforeinstallprompt` event (objected by Mozilla and WebKit)

Including any of those might potentially introduce a bias, which I'd like to avoid.

**Comment by andreasbm** [comment]
I think you have a really valid point. I agree that avoiding biases should be a priority for all of the resources. For the first iteration, I will create a "Proposals" skill in the "Modern Web" collection and move the proposals there as sub-skills. I will also change the descriptions to make it more clear when the skill is a proposal. In the future, I would love to create a "Proposals" collection and move them there.

Right now the skills are based on what I personally find to be the most useful on a day-to-day basis, but as the project grows and more and more people suggest resources, I'm going to need some clearer guidelines for what kind of resources should be on the list. I definitely think only adding API's that have positive signals from all major browser vendors should be one of the guidelines.

**Comment by web-padawan** [comment]
@andreasbm overall the idea sounds good to me 👍 

> I will create a "Proposals" skill in the "Modern Web" collection and move the proposals there as sub-skills.

Please note that in the "compact layout" mode, there is no visual indication what is a subskill, e.g. how Slots are related to Shadow DOM. 

So, I would appreciate a clear visual indicator for every proposal, e.g. a small icon, different border and text - anything that would mean "use at your own risk".

Another topic is that some articles like [Provide a custom install experience](https://web.dev/customize-install/) (describing `beforeinstallprompt` event) describing Chrome-only features should probably be removed.

**Comment by andreasbm** [comment]
Here's an update on my progress with this issue.

I've tested the solution of creating a proposals collection and I've tested moving the proposals to the "Modern Web" collection under a proposals skill. When testing both of these solutions I quickly noticed that I don't like when the skills are removed from their context (eg. not having "Form Participation" in the "Web Components" collection).

You mentioned having a visual indicator for every proposal which I tried and I think it works really well. The best solution I've ended up with so far is adding a banner to all of the skills that can be categorised as experimental and making it clear in the description that the skill experimental. 

![image](https://user-images.githubusercontent.com/6267397/79496565-9a0edf80-8026-11ea-8359-dd3f78481390.png)

![image](https://user-images.githubusercontent.com/6267397/79496514-86fc0f80-8026-11ea-9761-2384ac159eb8.png)

I thought about what definition we could use for "experimental" and decided to go with the definition from MDN which can be found [here](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Guidelines/Conventions_definitions#Experimental). I think the definition captures the meaning of experimental technology really well. I've added the following section to the README and to the help dialog.

> ### What does the "experimental" banner mean?
> 
> When a skill is described as experimental, it means that the technology is immature and currently in the process of being added to the Web platform (or considered for addition). Think carefully before you start using experimental technology in any kind of production project. The definition used in Web Skills is based on the excellent definition used on [MDN](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Guidelines/Conventions_definitions#Experimental). You are very welcome to open an issue if you see a skill you believe should be marked as "experimental" or have the banner removed.

That's my progress so far. I expect to have this live tomorrow 👍
